x-pack/plugins/timelines/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "author": "Elastic",
+  "name": "timelines",
+  "version": "8.0.0",
+  "private": true,
+  "license": "Elastic-License",
+  "scripts": {
+    "build-beat-doc": "node scripts/beat_docs/build.js && node ../../../scripts/eslint ./server/utils/beat_schema/fields.ts --fix"
```

Comment:
@rylnd would you be willing to consider keeping this script in the `security_solution` package for now, because we're planning some changes to the `timelines` plugin in `8.2`?

Reply:
@andrew-goldstein so do you want the script to generate an artifact within the `timelines` plugin, or do you want the `timelines` plugin to be updated to reference the artifact within the `security_solution` plugin? Happy to do either, just not sure which you're requesting.